### ISSUE TITLE
This example is not complete

### ISSUE THREAD

**Issue by kevando** [tommikaikkonen/redux-orm-example#1]
I really like this project so I want to help.

You probably know, but this example uses an older version of your API and it isn't quite complete in terms of linking all the selectors together. It also uses functions like `.toPlain()` that are no longer supported.

**Comment by tommikaikkonen** [comment]
Yes, this is sort of the old version of the introductory tutorial. I created a new one because I thought this one was too complex and detracted from learning how to use the library.

I'll add a deprecation notice to the README with a link to the more up to date `redux-orm-primer` repo.

Thanks!
